contrib/devtools/lint-python.sh

```diff
@@ -22,6 +22,7 @@
 # E275 missing whitespace after keyword
 # E304 blank lines found after function decorator
 # E306 expected 1 blank line before a nested definition
+# F401: module imported but unused
```

Comment:
This has snuck in somehow. Please remove! (F401 is documented lower down in the right place)

Reply:
Oh, thanks. Now fixed.